compiler/rustc_passes/src/lib_features.rs

```diff
@@ -54,6 +54,15 @@ impl<'tcx> LibFeatureCollector<'tcx> {
                         }
                     }
                 }
+                // We use ASCII escape codes so that the string does not get automatically replaced.
+                const VERSION_PLACEHOLDER: &str = "CURRENT_RUSTC_VERSI\x56N";
```

Comment:
I think typically we just handle this by excluding this file from handling -- I think this is likely to be almost worse since it makes it harder to find this place via e.g. grep, which isn't great.